ukraine and refugee expanders on domestic pages (#4445)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

    title: student finance
    text: |-
      <p>
      You’re unlikely to get help funding your training unless you have permission to <a href="https://www.gov.uk/browse/visas-immigration/settle-in-the-uk">live permanently in the UK</a>, have refugee status, or are training to teach languages or physics. You’ll also need to meet some other eligibility requirements.

>       <p>You may be eligible for financial support to train to teach if <a href="/non-uk-teachers/visas-for-non-uk-trainees"> you're from Ukraine or Afghanistan and on certain visas</a>.</p>

Reviewer: ```suggestion
      <p>You may be eligible for financial support to train to teach if <a href="/non-uk-teachers/visas-for-non-uk-trainees">you're from Ukraine or Afghanistan and on certain visas</a>.</p>
```

>       <p>If you're in <a href="https://www.gov.uk/claim-asylum"> the process of seeking asylum</a>, check your immigration bail conditions to see if you have permission to study on a teacher training course in England. Even if you have permission, you're unlikely to be eligible for financial support.

Reviewer: ```suggestion
      <p>If you're in <a href="https://www.gov.uk/claim-asylum">the process of seeking asylum</a>, check your immigration bail conditions to see if you have permission to study on a teacher training course in England. Even if you have permission, you're unlikely to be eligible for financial support.
```

>       <p>If you're granted asylum in the UK, you maybe eligible for financial support to train to teach.</p>

Author: ```suggestion
      <p>If you're granted asylum in the UK, you may be eligible for financial support to train to teach.</p>
```
